web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NetworkConfiguration.java

```diff
@@ -46,6 +47,9 @@
   @JsonProperty("neighbor_table")
   private List<NeighborEntry> neighborTable;
 
+  @JsonProperty("routerinfo")
+  private List<InternalRouterInfo> routerTable;
+
   public void addPortEntity(InternalPortEntity portEntity) {
     if (this.portEntities == null) {
       this.portEntities = new ArrayList<>();
```

Comment:
Need to add setter and getter for the new field, and update ToString method.

Reply:
fixed in a90538a